tasks/pipeline.py

```diff
@@ -420,6 +421,13 @@ def trigger_child_pipeline(_, git_ref, project_name, variables="", follow=True):
         print("Child pipeline finished successfully")
 
 
+@task
+def check_notify_teams(_):
+    if check_owners_slack():
+        Exit("Error: Some teams in CODEOWNERS don't have their slack notification channel specified !!", code=1)
```

Comment:
```suggestion
        Exit("Error: Some teams in CODEOWNERS don't have their slack notification channel specified in the GITHUB_SLACK_MAP !!", code=1)
```

Might allow people to know how to fix it fast